.azure-pipelines/windows.sh

```diff
@@ -51,15 +79,18 @@ clone_repos
 # Prepare build flags
 ################################################################################
 
-export CC="${MSVC_CC}"
-export AR="${MSVC_AR}"
-
 if [ "$MODEL" == "64" ] ; then
     export MODEL_FLAG="-m64"
     MAKE_FILE="win64.mak"
+    LIBNAME=phobos64.lib
+elif [ "$MODEL" == "32mscoff" ] ; then
+    export MODEL_FLAG="-m32mscoff"
+    MAKE_FILE="win64.mak"
+    LIBNAME=phobos32mscoff.lib
 else
     export MODEL_FLAG="-m32"
     MAKE_FILE="win32.mak"
+    LIBNAME=phobos.lib
```

Comment:
```suggestion
    LIBNAME=phobos32.lib
```

Reply:
phobos.lib is the name of the OMF library so far. That's why I had to special case it.